tutorials/editing-the-base-map/index.md

```diff
@@ -0,0 +1,465 @@
+---
+title: Editing the Base Map in Earth Engine
```

Comment:
Done; I have made a new directory on the same patch for the new tutorial. I am unsure whether I have to make a separate pull request or if it will work with this one.

Reply:
I don't see this change.

Reply:
Same here